__tests__/test.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Is this needed?

Reply:
Nahh good catch, will remove it. This was a temp file I made when I was debugging.